db/migrate/20151116114008_retire_links_column_from_links_sets.rb

```diff
@@ -0,0 +1,5 @@
+class RetireLinksColumnFromLinksSets < ActiveRecord::Migration
+  def change
+    rename_column :link_sets, :links, :legacy_links
+  end
+end
```

Comment:
Mmmh, I could check that the migration worked with a rake task that compares the old links with the new ones. Do you think it's necessary?